controllers/cloud.redhat.com/providers/kafka/managed-ephem.go

```diff
@@ -326,16 +326,29 @@ func (mep *managedEphemProvider) configureBrokers() error {
 	return nil
 }
 
-func (mep *managedEphemProvider) createCyndiPipeline(app *crd.ClowdApp) error {
+func (mep *managedEphemProvider) configCyndi(app *crd.ClowdApp) error {
 	if !app.Spec.Cyndi.Enabled {
 		return nil
 	}
-	return createCyndiPipeline(mep, app, getConnectNamespace(mep.Env), getConnectClusterName(mep.Env))
+
+	// TODO: as a followup, make these topic names configurable
+	eventsTopic := fmt.Sprintf("%s-platform.inventory.events", mep.Env.Name)
```

Comment:
Honestly, I'd move these two topic name strings into the other function, then the prototype becomes a lot simpler.

Reply:
Moved